internals/notifier.py

```diff
@@ -79,9 +91,13 @@ def format_email_body(
     new_val = prop['new_val']
     old_val = prop['old_val']
 
-    formatted_changes += ('<li><b>%s:</b> <br/><b>old:</b> %s <br/>'
-                          '<b>new:</b> %s<br/></li><br/>' %
+    # Call highlight_diff to get the highlighted differences
+    highlighted_new_val = highlight_diff(old_val, new_val)
```

Comment:
This produces a highlighted diff, but it is not highlighted quite the way that would be expected by people who are familiar with viewing diffs.   It puts all the highlights into the new value and nothing on the old value.  It looks like:
![image](https://github.com/GoogleChrome/chromium-dashboard/assets/17365/2bed9ae9-c112-4531-ab89-254dd6c17a2c)

It would be better to call highlight_diff() twice, once on the old value but highlighting only deletions, and once on the new value but highlighting only additions.  Then it could look like:
![image](https://github.com/GoogleChrome/chromium-dashboard/assets/17365/87c5f7ff-99e6-4e2b-8701-5c7b40ec28f2)



Also, you are not escaping the text that goes into or out of the differ.  That is a problem, because the input could include markup because our users discuss HTML tags, or if a malicious user tried to do something funny.  In this example, I changed the words "list item" to "<li>":
![image](https://github.com/GoogleChrome/chromium-dashboard/assets/17365/fc9ee7a4-1213-4413-a08e-a9b1f19721c5)

Reply:
It now separates the highlighting into two distinct calls for old and new values
It  escapes all text to prevent HTML injection, making the function safer